### DISCUSSION TITLE
Floodlight Not Working?

### DISCUSSION THREAD

**Discussion by jsiemonski** [mrlt8/docker-wyze-bridge#311] [Q&A]
Hello! I am on the latest version of the bridge and latest firmware on the camera, but I can't get my floodlight cam to work. It seems to break the camera in the Wyze app until I stop the Wyze Bridge. I tried using the Fresh Data switch, but it didn't help. It never seems to get past this point in the logs:

🚀 STARTING DOCKER-WYZE-BRIDGE v1.2.0
🏠 Home Assistant Mode
2022/02/28 11:32:55 [WyzeBridge] ♻️ FORCED REFRESH - Ignoring local 'user' data
2022/02/28 11:32:55 [WyzeBridge] ♻️ FORCED REFRESH - Ignoring local 'auth' data
2022/02/28 11:32:55 [WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
2022/02/28 11:32:55 [WyzeBridge] 💾 Saving 'auth' to local cache...
2022/02/28 11:32:55 [WyzeBridge] ☁️ Fetching 'user' from the Wyze API...
2022/02/28 11:32:56 [WyzeBridge] 💾 Saving 'user' to local cache...
2022/02/28 11:32:56 [WyzeBridge] ♻️ Refreshing camera data for thumbnails
2022/02/28 11:32:56 [WyzeBridge] ☁️ Fetching 'cameras' from the Wyze API...
2022/02/28 11:32:56 [WyzeBridge] 💾 Saving 'cameras' to local cache...
🎬 STARTING ALL 3 CAMERAS
2022/02/28 11:32:56 [WyzeBridge] Starting rtsp-simple-server v0.17.17
2022/02/28 11:32:56 [WyzeBridge] 🎉 Connecting to WyzeCam V3 - Driveway Cam on (IP) (1/3)
2022/02/28 11:32:56 [WyzeBridge] 🎉 Connecting to WyzeCam V2 - Den Cam on (IP) (1/3)
2022/02/28 11:32:56 [WyzeBridge] 🎉 Connecting to WyzeCam V2 - Camera 1 on (IP) (1/3)
2022/02/28 11:32:58 [Den Cam] 📡 Getting 120kb/s HD stream via LAN mode (WiFi: 100%) FW: 4.28.4.49 (2/3)
2022/02/28 11:32:58 [Driveway Cam] 📡 Getting 120kb/s HD stream via LAN mode (WiFi: 41%) FW: 4.36.8.26 🔒 (DTLS) (2/3)
2022/02/28 11:32:58 [Driveway Cam] WARNING: Skipping smaller frame at start of stream (frame_size=1)
2022/02/28 11:33:00 [Camera 1] 📡 Getting 120kb/s HD stream via LAN mode (WiFi: 100%) FW: 4.28.4.41 (2/3)
2022/02/28 11:33:03 [RTSP][DEN-CAM] ✅ '/den-cam' stream is UP! (3/3)
2022/02/28 11:33:08 [Den Cam] WARNING: Dropping old frames
2022/02/28 11:33:12 [Driveway Cam] AV_ER_DATA_NOREADY
2022/02/28 11:33:13 [WyzeBridge] 🎉 Connecting to WyzeCam V3 - Driveway Cam on 192.168.0.40 (1/3)
2022/02/28 11:33:17 [RTSP][CAMERA-1] ✅ '/camera-1' stream is UP! (3/3)
2022/02/28 11:33:25 [Driveway Cam] 📡 Getting 120kb/s HD stream via LAN mode (WiFi: 39%) FW: 4.36.8.26 🔒 (DTLS) (2/3)
2022/02/28 11:33:25 [Driveway Cam] WARNING: Dropping old frames

**Comment by mrlt8** [comment]
Interesting, I thought the floodlights were just regular v3s? I'll have to take a look to see if they made some changes in the app!

**Reply by jsiemonski** [reply]
It looks like a v3, but I’m guessing there are some differences. If you need me to poke around or grab logs, I can.

**Reply by mrlt8** [reply]
Can you get a steady stream in the app? Wifi looks kinda weak at 39%.

**Reply by jsiemonski** [reply]
Yes, the seconds counter increments smoothly.

**Comment by airbornetrooper82573** [comment]
I'm on beta firmware 4.36.8.32 on all my v3 cams, floodlight included. Wyze-bridge is up to date and everything is working for me.
